packages/strapi-design-system/src/Divider/Divider.tsx

```diff
@@ -0,0 +1,23 @@
+import styled from 'styled-components';
+
+import { Box, BoxProps } from '../Box';
+
+export interface DividerProps extends BoxProps {
```

Comment:
```suggestion
export interface DividerProps extends BoxProps<HTMLHRElement> {
```

Reply:
Also, because we override `background` & `as` maybe we should `omit` them from this type because if you pass them they won't take affect?

Reply:
Both good suggestions :) Applied them and rebased ✅